Group1/src/main/java/com/stuypulse/robot/commands/drivetrain/DrivetrainDriveCommand.java

```diff
@@ -0,0 +1,43 @@
+package com.stuypulse.robot.commands.drivetrain;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+import com.stuypulse.robot.subsystems.Drivetrain;
+import com.stuypulse.stuylib.input.Gamepad;
+import com.stuypulse.stuylib.math.SLMath;
+import com.stuypulse.stuylib.streams.filters.IFilter;
+import com.stuypulse.stuylib.streams.filters.LowPassFilter;
+import com.stuypulse.robot.constants.Settings;
+
+public class DrivetrainDriveCommand extends CommandBase {
+    private final Drivetrain drivetrain;
+    private final Gamepad gamepad;
+
+    private final IFilter turnFilter;
+    private final IFilter driveFilter;
+    
+    public DrivetrainDriveCommand(Drivetrain drivetrain, Gamepad gamepad) {
+        this.drivetrain = drivetrain;
+        this.gamepad = gamepad;
+
+        turnFilter = IFilter.create(
```

Comment:
We can pass in the joystick as a measurement to the filter already, so we don't need to do driveFilter.get(joystick stuff).